liquibase-standard/src/main/java/liquibase/command/core/CalculateChecksumCommandStep.java

```diff
@@ -47,42 +77,102 @@ public List<Class<?>> requiredDependencies() {
 
     @Override
     public String[][] defineCommandNames() {
-        return new String[][] { COMMAND_NAME };
+        return new String[][]{COMMAND_NAME};
     }
 
     @Override
     public void run(CommandResultsBuilder resultsBuilder) throws Exception {
         CommandScope commandScope = resultsBuilder.getCommandScope();
+
         final String changeSetIdentifier = commandScope.getArgumentValue(CHANGESET_IDENTIFIER_ARG);
         final String changeLogFile = commandScope.getArgumentValue(CHANGELOG_FILE_ARG).replace('\\', '/');
+        String changeSetPath;
+        String changeSetId;
+        String changeSetAuthor;
+
+        final boolean isChangeSetIdentifierPassed = changeSetIdentifier != null;
+
+        validateIdentifierParameters(commandScope, changeSetIdentifier);
```

Comment:
@JulienMa94 I extracted the parameters validation to a new method so it's easier to spot this change.